### PR TITLE
[ADD] remember_tree_column_width: Remember the tree columns' widths across sessions.

### PR DESCRIPTION
**Pull request by frahikLV**

### CODE REVIEW

**Comment by frahikLV** [comment]
@luisg123v Review this PR

File: remember_tree_column_width/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* None
```
Comment (luisg123v) [inline remember_tree_column_width/readme/ROADMAP.rst]: This is an optional file. If there's nothing to say, we may remove it

File: remember_tree_column_width/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Remember Tree Column Width",
+    "summary": "Remember the tree columns' widths across sessions.",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-ux",
+    "license": "LGPL-3",
+    "category": "Extra Tools",
+    "version": "15.0.1.0.0",
+    "maintainers": [
+        "frahikLV",
```
Comment (luisg123v) [inline remember_tree_column_width/__manifest__.py:10]: Could you add myself, too, please?

File: remember_tree_column_width/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "depends": [
+        "web",
+    ],
+    "data": [],
+    "assets": {
+        "web.assets_backend": [
+            "remember_tree_column_width/static/src/js/list_renderer.js",
+            "remember_tree_column_width/static/src/scss/style.scss",
```
Comment (luisg123v) [inline remember_tree_column_width/__manifest__.py]: maybe `main.scss`?

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by luisg123v** [comment]
Hi @carmenmiranda, @hugho-ad 

Could you review, please?

CC @desdelinux

**Comment by frahikLV** [comment]
@luisg123v Review

**Review by CarmenMiranda — APPROVED** [review]
Technically LGTM 👍🏼 

I would only recommend a little more description in the commit and description of the PR.

**Comment by pedrobaeza** [comment]
My concern is about the module name. A more standard `web_tree_column_width_remember` name would be advisable, as well as moving it to OCA/web

**Review by desdelinux — APPROVED** [review]
Functional Review is ok

Please, add a description to the PR

**Review by luisg123v — CHANGES_REQUESTED** [review]
LGTM, but I agree with @pedrobaeza, @CarmenMiranda and @desdelinux:

- Missing description
- Rename as suggested and move to OCA/web

**Pull request closed without merging**